CHANGELOG.md

```diff
@@ -62,6 +62,11 @@ For this reason, we are deprecating the monitoring port `tcp-prometheus` (9404)
 This port will be removed in the next release.
 Together with it we will also remove the Prometheus annotation from the service.
 
+#### Removal warning of Cluster Operator log level
+
+Because of the new Cluster Operator dynamic logging configuration via [PR#3328](https://github.com/strimzi/strimzi-kafka-operator/pull/3328) we are going to remove the `STRIMZI_LOG_LEVEL` environment variable from the Cluster Operator deployment YAML file in the next release.
+If you are using this environment variable in your current Cluster Operator installation, please takes this into account when updating to the next release.
```

Comment:
I would probably remove this sentence - It doesn't mean anything in 0.19.0 since nobody will remember now to do something in 0.20.0 based on this note. For that you need to make sure this is noted in #3328 ... It might just confuse people to thing the _next release_ referred here is 0.19.0.